content/docs/hooks-overview.md

```diff
@@ -99,9 +99,9 @@ function Example() {
 }
 ```
 
-When you call `useEffect`, you're telling React to run your "effect" function after flushing changes to the DOM. Effects are declared inside the component so they have access to its props and state. By default, React runs the effects after every render -- *including* the first render. (We'll talk more about how this compares to class lifecycles in [Using the Effect Hook](/docs/hooks-effect.html).)
+जब आप `useEffect` कॉल करते है, तो आप DOM के परिवर्तन फ्लश करने के बाद अपने "effect" फ़ंक्शन को चलाने के लिए React को बता रहे है। कौम्पोनॅन्टस के अंदर effects को डिक्लेअर किया जाता है ताकि उनके पास इसके props और state तक पहुंच हो। डिफ़ॉल्ट रूप से, React हर रेंडर के बाद effects चलाता है -- पहले रेंडर *के साथ भी* (हम इस बारे में और बात करेंगे की [Effect hook इस्तेमाल करते समय](/docs/hooks-effect.html) ये class lifecycles से कैसे तुलना करता है।)
 
-Effects may also optionally specify how to "clean up" after them by returning a function. For example, this component uses an effect to subscribe to a friend's online status, and cleans up by unsubscribing from it:
+Effects ऑप्शनल रूप से यह भी निर्दिष्ट कर सकते है कि किसी फ़ंक्शन को वापस करके उनके बाद उसका "clean up" कैसे करें। उदाहरण के लिए, यह कौम्पोनॅन्ट किसी मित्र की ऑनलाइन स्टेटस को सब्सक्राइब करने के लिए एक effect का उपयोग करता है, और बाद मैं इससे अनसब्सक्राइब करके क्लीन करता है:
```

Comment:
फ़ंक्शन को वापस करके उनके -> changes the contex, maybe leave return as it is and just change it in hindi and not translate?

Reply:
मित्र की -> ke

Reply:
by returning a function was lost

Reply:
@saranshkataria I have mentioned it in the next sentence, check it our here:
 https://github.com/reactjs/hi.reactjs.org/pull/58/commits/3d27ffc8a636e89130a75307af523125c2c251af#diff-9bda51acb98cffd656661def0255b05bR104